UICatalog/Scenarios/SplitContainerExample.cs

```diff
@@ -0,0 +1,149 @@
+﻿using Terminal.Gui;
+using System;
+using Terminal.Gui.Graphs;
+using NStack;
+using System.Linq;
+
+namespace UICatalog.Scenarios {
+	[ScenarioMetadata (Name: "Split Container", Description: "Demonstrates the SplitContainer functionality")]
+	[ScenarioCategory ("Controls")]
+	[ScenarioCategory ("LineView")]
+	public class SplitContainerExample : Scenario {
+
+		private SplitContainer splitContainer;
+
+		private SplitContainer splitContainer2;
+		private MenuItem miVertical;
+		private MenuItem miShowBoth;
+		private MenuItem miShowPanel1;
+		private MenuItem miShowPanel2;
+		private MenuItem miShowNeither;
+
+		/// <summary>
+		/// Setup the scenario.
+		/// </summary>
+		public override void Setup ()
+		{
+			// Scenario Windows.
+			Win.Title = this.GetName ();
+			Win.Y = 1;
+
+			Win.Add (new Label ("This is a SplitContainer with a minimum panel size of 4. Drag the splitter to resize:"));
+
+			splitContainer = new SplitContainer {
+				Y = 2,
+				X = 2,
+				Width = Dim.Fill () - 2,
+				Height = Dim.Fill () - 1,
+				SplitterDistance = Pos.Percent (50), // TODO: get this to work with drag resizing and percents
```

Comment:
Is this comment still relevant?

Reply:
Comment rot! Fixed in [675a6b0](https://github.com/gui-cs/Terminal.Gui/pull/2258/commits/675a6b02898b62a4cfbea637f9f6070ab750d3f3)